Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity.py

```diff
@@ -403,6 +453,58 @@ def get_alert_details_command(client: MsGraphClient, args):
     return hr, ec, alert_details
 
 
+def get_alert_details_v2_command(client: MsGraphClient, args):
+    alert_id = args.get('alert_id')
+    alerts_v2 = True
+
+    alert_details = client.get_alert_details(alert_id, alerts_v2)
+
+    hr = '## Microsoft Security Graph Alert Details - {}\n'.format(alert_id)
+
+    basic_properties_title = 'Basic Properties'
+    basic_properties = {
+        'ActorDisplayName': alert_details['actorDisplayName'],
```

Comment:
please use `get` for the other properties too and the context
```suggestion
        'ActorDisplayName': alert_details.get('actorDisplayName'),
```